### DISCUSSION TITLE
Почему отображается как троян?

### DISCUSSION THREAD

**Discussion by zajkagod** [bol-van/zapret#503] [General]
![image](https://github.com/user-attachments/assets/9ca37a92-ba5c-4e94-a331-8c84564b8b21)

скачал, вылезло такое, потом увидел что есть обсуждения на гитхабе, решил спросить

**Comment by testing765** [comment]
Чел ты в телеге скачал какую то фигню, конечно там может быть что угодно
Качай с нормального источника - https://github.com/bol-van/zapret-win-bundle/archive/refs/heads/master.zip

**Reply by zajkagod** [reply]
это файл отсюда, просто переслал его в диалоге в телеграме и перекачал заново

**Comment by bol-van** [comment]
Качать с первоисточника.
И иметь в виду, что дефендер может ругаться на пакованные upx бинарики nfqws/tpws
Для уверенности можно проверить на virustotal

**Reply by zajkagod** [reply]
это и есть с первоисточника, просто пересылка из диалога в диалог в телеграме)

**Reply by zajkagod** [reply]
на вирус тотале ругается deepinstinct

**Reply by zajkagod** [reply]
> Качать с первоисточника. И иметь в виду, что дефендер может ругаться на пакованные upx бинарики nfqws/tpws Для уверенности можно проверить на virustotal

![image](https://github.com/user-attachments/assets/3368ac77-7b6a-4b1a-9da7-8915b6166d85)
а это че

**Reply by bol-van** [reply]
Может быть антивирус его держит, чтобы не давать работать.
Если при отключении антивируса проблемы пропадают - разбирайтесь сами с вашими антивирусами

**Comment by vadimk1337** [comment]
Ты бы сначала прочитал quick_start_windows.txt 

"Имейте в виду, что антивирусы могут плохо реагировать на windivert.
cygwin имеет внушительный список несовместимостей с антивирусами. Многие антивирусы его ломают.
https://www.cygwin.com/faq.html#faq.using.bloda
Если это имеет место , используйте исключения. Если это не помогает - отключайте антивирус совсем."

**Reply by testing765** [reply]
Я тут проверил, и там не только на windivert ругается, но и на остальные бинарники, причем я сам попробовал скомпилировать исходник winws.exe и на него virustotal тоже ругается
Стремненько конечно

**Reply by bol-van** [reply]
Может потому, что он импортирует windivert.dll ?
